### PR TITLE
setup.py now remove unused pipeline steps

### PR DESCRIPTION
**Pull request by EugenioPetulla**
- Add wrapping comments to erasable steps in `.github/workflows/main.yml`
- Add a function to `setup.py` to delete the unused pipeline steps
- Exclude `venv` folder from delete step of `setup.py`

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -9,13 +9,39 @@
...
+
     repo_name = input("What is the name of your plugin (full name with spaces)? ")
 
     snake_name = "_".join(repo_name.lower().split(" "))
 
     os.rename("my_plugin.py", f"{snake_name}.py")
 
+    # Pipeline file's line ranges to remove
+    # This must reflect eventual main.yml changes
+    excluded_line_ranges = [(24, 35), (37, 40)]
```
Comment (zAlweNy26) [inline setup.py:39]: Isn't it better if we check by comments instead of by line ranges?

Comment (EugenioPetulla) [reply setup.py:39]: Why? What comment we should check for? Specific one? All the comments? What if we want to add other comments not related to the erasure task?

IMHO this was the simplest solution to the problem; less lines of code and not to complicate to read/understand.

Comment (zAlweNy26) [reply setup.py:39]: Ok chill bro, don't kill my family plz

Comment (EugenioPetulla) [reply setup.py:39]: I know your address! 🔪 🚿 😱 🩸

**Review by nicola-corbellini — APPROVED** [review]

**Pull request merged**